interfaces/Questioner.ts

```diff
@@ -0,0 +1,4 @@
+export interface Questioner {
+  QuestionaryName: string;
+  Rating: string;
+}
```

Comment:
If we had more time to do the CR, we could guess the meaning of Questioner & Questionary, but we don't so we don't know what you meant : (